scripts/Dockerfile.jenkins

```diff
@@ -2,21 +2,13 @@
 # so we'll have to hard code it here. Make sure to also update docker_version_tag when updating
 # the tag here.
 
-FROM swiftnav/buildroot-base:2019.03.15
+# TODO: replace with actual image tag once it's created
+FROM 571934480752.dkr.ecr.us-west-2.amazonaws.com/buildroot-base:test
```

Comment:
Assuming (per our discussion) that we'll change the docker-recipies repo to push this to docker hub?  Then we can purge the docker image building code from this repo.

Reply:
Changed to use dockerhub image

Reply:
This is still ECR?

Reply:
Oops, accidentally commented on old code.